actions/earned-income-budgeting.vbs

```diff
@@ -1482,39 +2500,43 @@ For ei_panel = 0 to UBOUND(EARNED_INCOME_PANELS_ARRAY, 2)       'looping through
-                                    'ADD ERROR HANDLING HERE
-                                    actual_checks_provided = TRUE
-                                End If
+                            '     If LIST_OF_INCOME_ARRAY(panel_indct, all_income) = ei_panel Then
+                            '         'ADD ERROR HANDLING HERE
+                            '         actual_checks_provided = TRUE
+                            '     End If
                             Next
+							save_your_work
                         End If
 
                         'If BOTH anticipated income AND actual checks are provided, the worker needs to chose which one should be budgeted.
                         If there_are_counted_checks = FALSE AND actual_checks_provided = TRUE Then
                             EARNED_INCOME_PANELS_ARRAY(pick_one, ei_panel) = use_estimate
+							save_your_work
                         ElseIf actual_checks_provided = TRUE AND anticipated_income_provided = TRUE Then
                             'CHOOSE CORRECT METHOD Dialog - select which (actual or anticipated) income information to budget and explain
                             Dialog1 = ""
                             BeginDialog Dialog1, 0, 0, 196, 165, "Reasonably Expected to Continue"
-                              OptionGroup RadioGroup1
-                                RadioButton 25, 70, 130, 10, "Use the actual check amounts/dates", use_actual_income
-                                RadioButton 25, 85, 130, 10, "Use the anticipated hours/wage", use_anticipated_income
-                              EditBox 10, 125, 180, 15, EARNED_INCOME_PANELS_ARRAY(selection_rsn, ei_panel)
-                              ButtonGroup ButtonPressed
-                                OkButton 140, 145, 50, 15
-                              Text 10, 10, 185, 35, "Both Actual Income and Anticipated Income have been listed for a SNAP case. Since both have been reported, both will be case ntoed. For entering information to the PIC, one option should be selected."
-                              GroupBox 5, 55, 185, 45, "Which is the best estimation of anticipated income?"
-                              Text 10, 110, 185, 10, "Explain why this is the best estimation of future income:"
+                              	OptionGroup RadioGroup1
+									RadioButton 25, 70, 130, 10, "Use the actual check amounts/dates", use_actual_income
+									RadioButton 25, 85, 130, 10, "Use the anticipated hours/wage", use_anticipated_income
+								EditBox 10, 125, 180, 15, EARNED_INCOME_PANELS_ARRAY(selection_rsn, ei_panel)
+								ButtonGroup ButtonPressed
+									OkButton 140, 145, 50, 15
+								Text 10, 10, 185, 35, "Both Actual Income and Anticipated Income have been listed for a SNAP case. Since both have been reported, both will be case ntoed. For entering information to the PIC, one option should be selected."
```

Comment:
2529: Spelling correction "both will be case ntoed" correct to "noted"

Reply:
Thank you!